guides/common/modules/proc_integrating-idm-dns-with-tsig-authentication.adoc

```diff
@@ -0,0 +1,64 @@
+[id="integrating-idm-dns-with-tsig-authentication"]
+= Integrating IdM DNS with TSIG authentication
+
+You can configure an IdM server to use the secret key transaction authentication for DNS (TSIG) technology that uses the `rndc.key` key file for authentication.
+The TSIG protocol is defined in https://tools.ietf.org/html/rfc2845[RFC2845].
+
+
+.Prerequisites
+
+* You must ensure the IdM server is deployed and the host-based firewall is configured correctly.
+ifdef::satellite[]
+For more information, see {RHELDocsBaseURL}7/html/linux_domain_identity_authentication_and_policy_guide/installing-ipa#prereq-ports[Port Requirements] in the _{RHEL}{nbsp}7 Linux Domain Identity, Authentication, and Policy Guide_.
+endif::[]
+* You must obtain `root` user access on the IdM server.
+* You must confirm whether {ProjectServer} or {SmartProxyServer} is configured to provide DNS service for your deployment.
+* You must configure DNS, DHCP and TFTP services on the base operating system of either the {Project} or {SmartProxy} that is managing the DNS service for your deployment.
```

Comment:
This isn't a prerequisite because at the end we point to a procedure to set this up
```suggestion
```